src/files/docker-essential/abstruse-pty.c

```diff
@@ -18,11 +18,15 @@ int main(int argc, char *argv[]) {
     perror("fork");
   } else if (pid == 0) { // child
     char *args[] = { NULL };
+    dup2(fd[0], STDIN_FILENO);
```

Comment:
Processes in CI complain that `is not a TTY` because there is no `stdin`. Changed so `stdin`1 is not closed on forked processes.